src/compiler/CompileContext.js

```diff
@@ -137,6 +137,7 @@ class CompileContext extends EventEmitter {
         this.compilerVersion = this.options.compilerVersion || markoPkgVersion;
         this.writeVersionComment = writeVersionComment !== 'undefined' ? writeVersionComment : true;
         this.ignoreUnrecognizedTags = this.options.ignoreUnrecognizedTags || false;
+        this.escapeAtTags = this.options.escapeAtTags || false;
```

Comment:
Oh, good catch. Sorry about that.
Let me actually use `this.escapeAtTags` on lines 464 and 479.